doc/rst/usingchapel/compiling.rst

```diff
@@ -50,9 +50,9 @@ We note some of the most useful flags here:
 =====================   ======================================================
 Flags                   Description
 ---------------------   ------------------------------------------------------
-``-o <filename>``       specify the filename of the generated
-                        executable, otherwise the main module name is
-                        used by default
+``-o <filename>``       specify the filename of the generated executable,
+                        otherwise the filename of the main module (minus
```

Comment:
'filename of the main module' seems strange to me, but what about 'name of the file containing the main module' ?